### PR TITLE
[master]update doc for deleting the resize code when creating image

### PR DESCRIPTION
**Pull request by wgxoyun**
Signed-off-by: wgxoyun <wgxoyun@cn.ibm.com>

### CODE REVIEW

**Comment by bjhuangr** [comment]
Are you sure this doc change related to deleting resize code?

**Comment by wgxoyun** [comment]
Yes, before the deletion, zcc uses resize command to resize disk, after deletion, we use cloud-init to resize disk, so zcc will depend on cloud init. So i change the image creation document to tell user the cloud-init is necessary. @bjhuangr

File: doc/source/makeimage.rst [hunk 1]
```diff
@@ -148,27 +148,24 @@ If above commands execute successfully, you may continue to next steps.
...
 
 
-Configuration of activation engine(AE) in zLinux
+Configuration of cloud-init in zLinux
 ------------------------------------------------
 To do useful work with the user data, the zLinux image must be configured to
 run a service that retrieves the user data passed from the z/VM Cloud Connector
-and then takes some actions based on the contents of that data. This service is
-also known as an activation engine (AE).
+and then takes some actions based on the contents of that data. This function can
```
Comment (bjhuangr) [inline doc/source/makeimage.rst:155]: function -> task

File: doc/source/makeimage.rst [hunk 2]
```diff
@@ -148,27 +148,24 @@ If above commands execute successfully, you may continue to next steps.
...
+These steps of configuration zvmguestconfigure and cloud-init are described in subsequent sections.
 
 Configuration of zvmguestconfigure in zLinux
 --------------------------------------------
 The zvmguestconfigure script/service must be installed in the zLinux so it
 can process the request files transmitted by z/VM Cloud Connector to the
 reader of the zLinux as a class X file. zvmguestconfigure also act as the bridge
 between the zLinux and higher layer of zVM Cloud. Take spawning a VM via Openstack
-nova-zvm-driver for example, the image use cloud-init as the underlying AE.
+nova-zvm-driver for example, the image use cloud-init.
```
Comment (bjhuangr) [inline doc/source/makeimage.rst:168]: use -> uses

**Comment by bjhuangr** [comment]
@dongyanyang please double check.

**Review by dongyanyang — APPROVED** [review]

**Pull request merged**